tests/components/application-connector/README.md

```diff
@@ -5,19 +5,35 @@ These are the component tests for Application Connector.
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
 **Table of Contents**
 
-- [Design and architecture](#design-and-architecture)
-- [Mock application](#mock-application)
-- [Building](#building)
-- [Running](#running)
-    - [Deploy a Kyma cluster locally](#deploy-a-kyma-cluster-locally)
-    - [Run the tests](#run-the-tests)
-- [Debugging](#debugging)
-    - [Running locally](#running-locally)
-    - [Running without cleanup](#running-without-cleanup)
+- [Component tests for Application Connector](#component-tests-for-application-connector)
+   - [Application Gateway](#application-gateway)
+      - [Design and architecture](#design-and-architecture)
+      - [Mock application](#mock-application)
+         - [Certificates](#certificates)
+         - [API exposed on port `8080`](#api-exposed-on-port-8080)
+         - [API exposed on port `8090`](#api-exposed-on-port-8090)
+         - [API exposed on port `8091`](#api-exposed-on-port-8091)
+      - [Building](#building)
+      - [Running](#running)
+         - [Deploy a Kyma cluster locally](#deploy-a-kyma-cluster-locally)
+         - [Run the tests](#run-the-tests)
+      - [Debugging](#debugging)
+         - [Running locally](#running-locally)
+         - [Running without cleanup](#running-without-cleanup)
+   - [Application Connectivity Validator](#application-connectivity-validator)
+       - [Design and architecture](#design-and-architecture-1)
+      - [Building](#building-1)
+      - [Running](#running-1)
+         - [Deploy a Kyma cluster locally](#deploy-a-kyma-cluster-locally-1)
+         - [Run the tests](#run-the-tests-1)
+      - [Debugging](#debugging-1)
+         - [Running without cleanup](#running-without-cleanup-1)
```

Comment:
```suggestion
- [Component tests for Application Connector](#component-tests-for-application-connector)
   - [Application Gateway](#application-gateway)
      - [Design and architecture](#design-and-architecture)
      - [Mock application](#mock-application)
         - [Certificates](#certificates)
         - [API exposed on port `8080`](#api-exposed-on-port-8080)
         - [API exposed on port `8090`](#api-exposed-on-port-8090)
         - [API exposed on port `8091`](#api-exposed-on-port-8091)
      - [Building](#building)
      - [Running](#running)
         - [Deploy a Kyma cluster locally](#deploy-a-kyma-cluster-locally)
         - [Run the tests](#run-the-tests)
      - [Debugging](#debugging)
         - [Running locally](#running-locally)
         - [Running without cleanup](#running-without-cleanup)
   - [Application Connectivity Validator](#application-connectivity-validator)
       - [Design and architecture](#design-and-architecture-1)
      - [Building](#building-1)
      - [Running](#running-1)
         - [Deploy a Kyma cluster locally](#deploy-a-kyma-cluster-locally-1)
         - [Run the tests](#run-the-tests-1)
      - [Debugging](#debugging-1)
         - [Running without cleanup](#running-without-cleanup-1)
```

The [governance job](https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/14912/pre-main-kyma-governance/1564137731880128512)'s failing because it doesn't recognize the links with `-1` at the end, but the links are working. 
Unfortunately, I believe this job is required to pass. 
We cannot remove the `-1` from the end of the links because then we're linking to stuff for Application Gateway instead of Connectivity Validator. 
We have two options:
- Create two separate "READMEs" (probably best to call them something else, like `application-gateway-tests.md` and `application-connectivity-validator-tests.md` and link to them from the README file (this seems the cleanest solution). 
- Add a prefix/suffix (`AG`/`ACV`) to the section titles for each component, so that they're different and there are no conflicts in linking, and then update the TOC. 

For example: 
```
## Application Gateway

### Design and architecture (AG)
```

or 

```
## Application Gateway

### Design and architecture [AG]
```

or 

```
## Application Gateway

### [AG] Design and architecture 
```
or something like this.